time_bucket_ng() - document time zones support (#383)
timescale/docs: api/time_bucket_ng.md

## timescaledb_experimental.time_bucket_ng() <tag type="experimental">Experimental</tag>
The `time_bucket_ng()` (next generation) experimental function is an updated
version of  the original [`time_bucket()`][time_bucket] function. While

> `time_bucket` works only with small units of time, the `time_bucket_ng()`

Reviewer: ```suggestion
`time_bucket` works only with small units of time,  `time_bucket_ng()` 
```

 2021-08-23 00:00:00
```

> The following example shows how `time_bucket_ng()` can be used to bucket data

Author: 👍

> The following example shows how `time_bucket_ng()` can be used to bucket data
> by months in specified timezone:

Reviewer 2: ```suggestion
This example shows how `time_bucket_ng()` is used to bucket data
by months in a specified timezone:
```
